src/VisualStudio/Core/Def/Implementation/ProjectSystem/AbstractEncProject_IVsENCRebuildableProjectCfg.cs

```diff
@@ -20,57 +19,58 @@ public int HasCustomMetadataEmitter(out bool value)
 
         public int StartDebuggingPE()
         {
-            return EditAndContinueImplOpt.StartDebuggingPE();
+            return EditAndContinueImplOpt != null ? EditAndContinueImplOpt.StartDebuggingPE() : VSConstants.S_OK;
         }
 
         public int StopDebuggingPE()
         {
-            return EditAndContinueImplOpt.StopDebuggingPE();
+            return EditAndContinueImplOpt != null ? EditAndContinueImplOpt.StopDebuggingPE() : VSConstants.S_OK;
         }
 
         public int GetPEidentity(Guid[] pMVID, string[] pbstrPEName)
         {
-            return EditAndContinueImplOpt.GetPEidentity(pMVID, pbstrPEName);
+            return EditAndContinueImplOpt != null ? EditAndContinueImplOpt.GetPEidentity(pMVID, pbstrPEName) : VSConstants.S_OK;
```

Comment:
Especially this one - pMVID and pbstrPEName are expected to be filled with values.